sw/host/opentitanlib/src/io/emu.rs

```diff
@@ -30,9 +32,33 @@ impl_serializable_error!(EmuError);
 pub enum EmuValue {
     Empty,
     String(String),
-    FilePath(String),
+    FilePath(PathBuf),
     StringList(Vec<String>),
-    FilePathList(Vec<String>),
+    FilePathList(Vec<PathBuf>),
+}
+
+impl From<String> for EmuValue {
+    fn from(s: String) -> Self {
+        EmuValue::String(s.clone())
```

Comment:
Aren't the `clones()`s here and below redundant, as `String`, `PathBuf`, and `Vec` all have ownership of their underlying data?

Reply:
Done